.env.example

```diff
@@ -1,5 +1,7 @@
-MUMBAI_RPC_URL="example: https://rpc-mainnet.maticvigil.com      NOTE: set MUMBAI_RPC_URL or GOERLI_RPC_URL, but not both.  Delete one of these lines"
-GOERLI_RPC_URL="example: https://goerli.infura.io/v3/1234567890  NOTE: set MUMBAI_RPC_URL or GOERLI_RPC_URL, but not both.  Delete one of these lines"
+# NOTE: set only one RPC_URL, comment or delete others
+MUMBAI_RPC_URL="example: https://rpc-mainnet.maticvigil.com"
+GOERLI_RPC_URL="example: https://goerli.infura.io/v3/1234567890"
+SEPOLIA_RPC_URL="check infra for RPC url"
```

Comment:
We don't need all those examples, just use "[UNSET]" for all or some other marker.

Reply:
We should not set these values to anything if they are not needed.  Just leave them as `undefined`.